docs/api/createNavLinkComponent.md

```diff
@@ -0,0 +1,25 @@
+# createNavLinkComponent
+
+`createNavLinkComponent` is a function for you to create a NavLink component, you are able to pass options to overwrite attributes to generate the NavLink. e.g., custom mixin or click handler.
+
+## Parameters
+
+| Param Name | Param Type | Description |
+|-----------|-----------|-------------|
+| options | Object | the object taken to overwrite the default attribute when we create NavLink |
```

Comment:
sure updated, also update the parameter name, use `overwriteSpec`, which align with React, https://github.com/facebook/react/blob/master/src/isomorphic/classic/class/ReactClass.js#L804